go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	github.com/che-incubator/kubernetes-image-puller-operator v0.0.0-20210929175054-0128446f5af7
 	github.com/devfile/api/v2 v2.2.2
-	github.com/devfile/devworkspace-operator v0.25.0
+	github.com/devfile/devworkspace-operator v0.29.0
```

Comment:
Please update DEPENDENCIES.md file with a new DWO version and link to https://clearlydefined.io/